src/io/flutter/dart/DartSyntax.java

```diff
@@ -70,21 +70,20 @@ public static DartCallExpression findEnclosingFunctionCall(@NotNull PsiElement e
    *
    * @return true if the given element is a call to function, false otherwise
    */
-  public static boolean isCallToFunctionNamed(@Nullable PsiElement element, @NotNull String function) {
-    if (!(element instanceof DartCallExpression)) return false;
-    final String name = getCalledFunctionName((DartCallExpression)element);
+  public static boolean isCallToFunctionNamed(@NotNull DartCallExpression element, @NotNull String function) {
+    final String name = getCalledFunctionName(element);
     return Objects.equals(name, function);
   }
 
   /**
-   * Check if an element is a declaration of a function with the given name.
+   * Check if an element is a declaration of "main".
    *
-   * @return true if the given element is a declaration, false otherwise
+   * @return true if the given element is a main declaration, false otherwise
    */
-  public static boolean isFunctionDeclarationNamed(@Nullable PsiElement element, @NotNull String name) {
+  public static boolean isMainFunctionDeclaration(@Nullable PsiElement element) {
     if (!(element instanceof DartFunctionDeclarationWithBodyOrNative)) return false;
     final String functionName = ((DartFunctionDeclarationWithBodyOrNative)element).getComponentName().getId().getText();
-    return Objects.equals(functionName, name);
+    return Objects.equals(functionName, "main");
```

Comment:
In general we haven't been creating static fields for Strings that will just get interned (which is just to say there shouldn't be any significant performance difference).  As to whether it'd be clearer, I used to agree but now tend to think 

    Objects.equals(functionName, "main")

is clearer than

    Objects.equals(functionName, MAIN_FUNC_NAME)

where we have to define

    static final String MAIN_FUNC_NAME = "main";

elsewhere.